docs/tutorial/common_hpc_dependencies.rst

```diff
@@ -167,8 +167,9 @@ compilers be set as the main compilers. This will change soon.
 **Important Setup Variables**
 
 * ``ENABLE_HIP`` : Enables HIP support in BLT
-* ``HIP_ROOT_DIR`` : Root directory for HIP installation
+* ``ROCM_ROOT_DIR`` : Root directory for ROCM installation
 * ``CMAKE_HIP_ARCHITECTURES`` : GPU architecture to use when generating HIP/ROCm code
+* ``CMAKE_HIP_COMPILER``: Compiler to use for HIP compilation, required when the C++ compiler does not support HIP.
```

Comment:
I think this statement is a little confusing. I thought if you use HIP with BLT after this pull request, CMAKE_HIP_COMPILER is always required to be specified. I would leave off everything after the comma.

Reply:
Not required, take a look at the host configs.

Reply:
Is it required if you use CMAKE_HIP_FLAGS?

Reply:
```
$ cmake -DCMAKE_HIP_FLAGS="-g" -DENABLE_HIP=On ../tests/internal
-- The C compiler identification is Clang 18.1.8
-- The CXX compiler identification is Clang 18.1.8
<snip>
-- BLT HIP support is On
-- Creating BLT HIP targets...
-- The HIP compiler identification is Clang 18.0.0
-- Detecting HIP compiler ABI info
-- Detecting HIP compiler ABI info - done
-- Check for working HIP compiler: /opt/rocm-6.2.1/lib/llvm/bin/clang++ - skipped
```

Reply:
Great!